src/helpers/formatFilterNames.js

```diff
@@ -0,0 +1,16 @@
+import { options } from 'config'
+import formatString from 'helpers/formatString'
+import formatPlatformName from 'helpers/formatPlatformName'
+import formatTechnologyName from 'helpers/formatTechnologyName'
+
+export default (filterOption, item, formattedPlatformName) => {
+  const { filterOptionsToFormat } = options.search
+  const formatName = () =>
+    filterOption === 'platforom'
+      ? formatPlatformName(formattedPlatformName)
+      : formatTechnologyName(item)
+
+  return !filterOptionsToFormat[filterOption]
+    ? formatString(item)
+    : formatName()
+}
```

Comment:
I think we can make this easier to understand. This function really just needs to take a `filterOption` and map that to a corresponding formatting function. 

This means that we will need to move the cache.platforms logic to the `formatPlatformName` helper. That function should still just take `platformName` as the only argument only now it will check to see if it is in the cache then handle the value the same as it currently does.

```suggestion
export default (filterOption, name) => {
  
  if (filterOption === 'platform') {
    return formatPlatformName(name)
  }
  
  if (filterOption === 'technology') {
    return formatTechnologyName(name)  
  }

  return formatString(item)
}
```

Reply:
Now using the API cache directly in `helpers/formatPlatformName` and refactored `helpers/formatFilterName` with the switch statement for readability and maintainability. Thank you for the feedback!